### ISSUE TITLE
test-pattern does not seem to be working

### ISSUE THREAD

**Issue by vanniktech** [arturbosch/detekt#513]
I've got the following configuration:

```yml
test-pattern: # Configure exclusions for test sources
  active: true
  patterns: # Test file regexes
    - '.*/test/.*'
    - '.*Test.kt'
    - '.*Spec.kt'
  exclude-rule-sets:
    - 'comments'
  exclude-rules:
    - 'NamingRules'
    - 'WildcardImport'
    - 'MagicNumber'
    - 'MaxLineLength'
    - 'LateInitUsage'
    - 'StringLiteralDuplication'
    - 'SpreadOperator'
    - 'TooManyFunctions'
    - 'LargeClass'
```

which is the default configuration and should be active on it's own, especially when used with `failFast`, right?

When I run `detektCheck` with RC5, I get the following report:

```
LateinitUsage - [keyboardHandler] at app/src/test/java/com/vanniktech/chessclock/android/keyboard/AndroidKeyboardHandlerTest.kt:27:3
```

which should not be reported since first of all the file naming matches and also the `test` directory. Could this be somehow related to #510 where the filepath would need to be resolved too?

**Comment by arturbosch** [comment]
Looks like I spelled `LateInitUsage` wrong in the config. It reports `LateinitUsage` :D
